README.md

```diff
@@ -50,3 +51,8 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Answer
```

Comment:
Apart from that, we can't indicate time complexity as n^2 + n! -- In polynomial, you have to specify which one is asymptotically bigger. n! = n(n-1)(n-2)...1, which approximately as same as n^n